device/api/umd/device/cluster.h

```diff
@@ -698,15 +703,42 @@ class Cluster : public tt_device {
      * @param simulated_harvesting_masks
      */
     Cluster(
+        tt_SocDescriptor soc_desc,
```

Comment:
Figure out what happens with harvesting now. Previously harvesting was passed with cluster descriptor. Then we moved to using default cluster descriptor. But now, that the object is passed, who does the harvesting?

Reply:
Synced offline with Pavle. 
Will wait for his changes which remove harvesting from cluster.cpp class and start using tt_socdescriptor's harvesting

Reply:
After more discussion offline, I've reverted this change. We don't want to allow the user to have an option to pass wrong harvesting info.